rust/src/action/parquet2_read/dictionary/binary.rs

```diff
@@ -0,0 +1,49 @@
+use parquet2::encoding::get_length;
+use parquet2::error::Error;
+
+// FIXME: replace vec with reference
```

Comment:
should this be fixed in this PR?

Reply:
good catch, totally forgot about this one.